### DISCUSSION TITLE
Will this project provide static library?

### DISCUSSION THREAD

**Discussion by calvin2021y** [c-smile/sciter-js-sdk#160] [General]
Will this project provide static library?

**Comment by c-smile** [comment]
scapp[.exe] is built using static Sciter library - it does not need sciter.dll to run.

But I am not publishing Sciter static library as it is not realistic logistically. You can use MSVC, GCC, LLVM to compile your project and so you need corresponding Sciter's static lib. As also architectures: x86, x64, ARM. As also runtime libs.

Too many possible variations. Normally customers are building static Sciter from sources for particular architectures they use.
